docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.icons.scss

```diff
@@ -136,62 +136,32 @@
   }
 }
 
-@mixin hb-icon-link-arrow($color: 'black') {
+@mixin hb-icon-link-arrow($color: 'black', $drop-shadow: false) {
```

Comment:
@mariannuar For consistency with the original mixin, let's change the variable name to `$box-shadow` please.